### DISCUSSION TITLE
peterkimzz: %EA%B8%B0%EC%88%A0%EA%B3%BC %EC%98%88%EC%88%A0%EC%9D%84 %EB%8B%B4%EC%9D%80 %EB%B8%94%EB%A1%9C%EA%B7%B8

### DISCUSSION THREAD

**Discussion by giscus** [peterkimzz/blog#26] [Blog Comments]
# peterkimzz: %EA%B8%B0%EC%88%A0%EA%B3%BC %EC%98%88%EC%88%A0%EC%9D%84 %EB%8B%B4%EC%9D%80 %EB%B8%94%EB%A1%9C%EA%B7%B8

https://www.peterkimzz.com/nuxt3-sideproject-2/

**Comment by digitalsmh** [comment]
안녕하세요.  Nuxt3 개발환경, 프로젝트 적용 등등 구경하러 왔습니다. 잘 부탁드립니다.^^
